evalset/multiopt_test_funcs.py

```diff
@@ -0,0 +1,25 @@
+from evalset.test_funcs import TestFunction, lzip
+import numpy
+
+class MultioptTestFunction(TestFunction):
+
+  def __init__(self, dim):
+    super(MultioptTestFunction, self).__init__(dim)
+    self.local_minima_loc = [] # Sorted in increasing order of function value at the local minima
+
+class LowDMixtureOfGaussians(MultioptTestFunction):
```

Comment:
Are you going to be adding more of these?  If so, you might want to append 01 to this.  Or maybe make this an extensible class.  If it'll be a lot of different functions then it's all good.